careers/students-and-graduates/index.html

```diff
@@ -0,0 +1,221 @@
+{% extends "layout-side-nav.html" %}
+{% import "_vars-careers.html" as vars with context %}
+{%- set active_nav_id = 'students-and-graduates' -%}
+
+{% block title -%}
+    Students and Graduates
+{%- endblock %}
+
+{% block desc -%}
+
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom
+{%- endblock %}
+
+{% block content_main %}
+
+    <section class="media block block__flush-top block__sub">
+
+            <h1>Students and graduates</h1>
+            <p class="h3">
+                The CFPB is committed to creating and sustaining
+                a new generation of leaders.
+                There are many opportunities for you to make an immediate impact.
+            </p>
+    </section>
+
+    <section class="students-and-graduates">
```

Comment:
For working with the CFPB I ended up putting this in the `content_modifiers` block to wrap the whole content area and prefixing it with `careers-`. What do you think about removing this here and adding the following after the css block?

```jinja
{% block content_modifiers -%}
    {{ super() }}
    careers-students-and-graduates
{%- endblock %}
```